<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zhanghao</class>
 <widget class="QDialog" name="zhanghao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/C:/Users/Administrator.QH-20141214JPFU/Pictures/0]48MKQOOPNTW%60}V2NM]L.png</normaloff>:/new/prefix1/C:/Users/Administrator.QH-20141214JPFU/Pictures/0]48MKQOOPNTW%60}V2NM]L.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="newlineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zh_queren">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newlineEdit_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldlineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>原密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zh_fanhui">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>461</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/38.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>newlineEdit_2</zorder>
  <zorder>label_3</zorder>
  <zorder>zh_queren</zorder>
  <zorder>newlineEdit_3</zorder>
  <zorder>label_2</zorder>
  <zorder>oldlineEdit</zorder>
  <zorder>label</zorder>
  <zorder>zh_fanhui</zorder>
 </widget>
 <tabstops>
  <tabstop>oldlineEdit</tabstop>
  <tabstop>newlineEdit_2</tabstop>
  <tabstop>newlineEdit_3</tabstop>
  <tabstop>zh_queren</tabstop>
  <tabstop>zh_fanhui</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
